github-config.yaml

```diff
@@ -136,6 +136,10 @@ orgs:
           by our bots...
         has_projects: false
         has_wiki: false
+      data-driven-development:
+        description: ""
```

Comment:
can you please add a small description: `This repository consolidates and provides resources, guides and documentation necessary to foster data driven development.`

Reply:
@hemajv done :+1: